### PR TITLE
Cleaned code, Added logger, Fixed string decryptor, Added better command line parser.

### PR DESCRIPTION
**Pull request by PuffingIn2D**

### CODE REVIEW

File: EazFixer.Code/App.config [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (HoLLy-HaCKeR) [inline EazFixer.Code/App.config]: I do not think that splitting up the code into 2 projects is needed. If certain logic has to be usable as a library, it should just be made public so others can reference the executable.

File: EazFixer.Code/Entrypoint.cs [hunk 2]
```diff
@@ -0,0 +1,87 @@
+using System.IO;
+using System.Linq;
+using EazFixer.Code.Other;
+using EazFixer.Code.Processors;
+
+namespace EazFixer.Code {
```
Comment (HoLLy-HaCKeR) [inline EazFixer.Code/Entrypoint.cs]: Please follow the original code style of the project. In particular, brackets should not be placed on the same line unless for if-, for- or using-statements.

File: EazFixer.Code/Entrypoint.cs [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    public class Entrypoint {
+        public static int Start(string[] args) {
+            Logger.Initialize("EazFixerLog.md", "[~]");
```
Comment (HoLLy-HaCKeR) [inline EazFixer.Code/Entrypoint.cs]: Is logging really needed? Only a few lines should be printed out, which could easily be done in the console.

File: EazFixer.Code/Other/Logger.cs [hunk 4]
```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.IO;
+
+namespace EazFixer.Code.Other {
+    internal class Logger {
+        private static StreamWriter stream_;
```
Comment (HoLLy-HaCKeR) [inline EazFixer.Code/Other/Logger.cs]: Why does this have a _ suffix? 😕 Please follow the naming style of the rest of project.

**Comment by PuffingIn2D** [comment]
Ok sorry, I will make another fork and only fix the issues and submit anotjer request, I will keep my fork a separate version of this program.

**Pull request closed without merging**